openstack/02b-MAChigh/schedule.c

```diff
@@ -178,6 +179,15 @@ void schedule_setFrameLength(frameLength_t newFrameLength) {
    ENABLE_INTERRUPTS();
 }
 
+void schedule_setCustomFrameLength(frameLength_t newFrameLength) {

```

Comment:
`schedule_setFrameLength` is the default function provided by schedule component. I remove the schedule_setCustomFrameLength  function and using schedule_setFrameLength to do this job. See: https://github.com/changtengfei/openwsn-fw/commit/555cee397933954044a9dc524293f1d2fe45283e